tsconfig.json

```diff
@@ -24,7 +25,12 @@
     "indentSize": 2,
     "tabSize": 2
   },
-  "files": [
-    "src/Rx.ts"
+  "include": [
+    "src"
```

Comment:
let's split it. Especially current tsconfig.json is not being used actively (npm script runs command for its own).